src/main/resources/org/sagebionetworks/web/client/widget/entity/EntityMetadataViewImpl.ui.xml

```diff
@@ -6,11 +6,14 @@
   xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
   xmlns:w="urn:import:org.sagebionetworks.web.client.widget"
 >
-  <g:HTMLPanel>
+  <g:HTMLPanel addStyleNames="entityMetadata entity-page-side-margins">
     <g:HTMLPanel ui:field="detailedMetadata">
       <w:ReactComponentDiv ui:field="annotationsModalContainer" />
       <bh:Div>
-        <bh:Span addStyleNames="boldText margin-right-5" text="Synapse ID:" />
+        <bh:Span
+          addStyleNames="boldText margin-right-5"
+          text="Project SynID:"
```

Comment:
This section is now only shown for projects, so we clarify that this is the Project SynID (specified by design)